### PR TITLE
chore: add template README and tweak setup.py

### PR DESCRIPTION
**Pull request by busunkim96**
- Add a license header to the empty `__init__.py` in the unit test directory. (OSPO's tool flags this as missing a license header.)
- Teak the setup.py so there are fewer manual adjustments needed during initial repo setup of a Cloud library.
- Add a generic README  with instructions on how to install the library locally. This is for the folks using the self-service approach. (A README for a published library needs more content like links to PyPI, links to documentation, etc.).

### CODE REVIEW

**Comment by busunkim96** [comment]
Friendly ping @software-dov

File: gapic/templates/setup.py.j2 [hunk 1]
```diff
@@ -24,9 +34,6 @@ setuptools.setup(
     {%- endif %}
     ),
     python_requires='>=3.6',
-    scripts=[
-        'scripts/fixup_{{ api.naming.versioned_module_name }}_keywords.py',
-    ],
```
Comment (software-dov) [inline gapic/templates/setup.py.j2:29]: Does the PEP420PackageFinder autopopulate this in packages?

File: gapic/templates/setup.py.j2 [hunk 2]
```diff
@@ -1,17 +1,27 @@
 {% extends '_base.py.j2' %}
 
 {% block content %}
+import io
```
Comment (software-dov) [inline gapic/templates/setup.py.j2:4]: Just curious, why use `io.open` instead of builtin `open`?

**Review by software-dov — APPROVED** [review]
LGTM, just two questions.

**Pull request merged**